### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nblackburn**
Updated to follow new Laravel conventions.

### CODE REVIEW

**Comment by vinkla** [comment]
Thanks for you contribution. I haven't made up my mind about these yet. I'll get back to you when I do.

**Comment by GrahamCampbell** [comment]
I'm :-1: on this change. I think it was stupid for laravel/laravel to change the config to use ::class.

**Comment by nblackburn** [comment]
@GrahamCampbell Not sure myself but that's what they are using so it makes sense to keep conventions.

**Comment by nblackburn** [comment]
Upon further investigation it seems the new ``::class`` doesn't perform any checks to the existence of the targeted class and thus doesn't raise any errors which i guess could be a problem.

**Comment by GrahamCampbell** [comment]
I think it doesn't make any sense in config. Fine for your actual code.

**Comment by GrahamCampbell** [comment]
> so it makes sense to keep conventions.

Not if people don't use ::class in their config files. It's hardly a hardship for people to just use ::class anyway from the example in the readme.

**Comment by nblackburn** [comment]
Fair point, completely agree with you on that one.

**Comment by vinkla** [comment]
Though, newcomers and new installations of Laravel this would help a lot. They probably wont use string in their config. They'll use what comes out of the box.

**Comment by GrahamCampbell** [comment]
Yeh, but us telling them what class name to use doesn't mean they can't use ::class.

**Comment by GrahamCampbell** [comment]
Maybe we need to reword this to not say "add this to the config", but instead, register this service provider.

**Comment by vinkla** [comment]
That could be one way to go. Though, new users will see that this differs from core and it could become a problem for them not understanding how this should be implemented.

**Comment by nblackburn** [comment]
@vinkla Pretty much my thinking and why i proposed the change.

**Comment by vinkla** [comment]
Thanks for proposing this @nblackburn but after some consideration I've decided to stick with the string. Personally I prefer it over `::class` notation.
